docs/developer/building/requirements.rst

```diff
@@ -0,0 +1,32 @@
+Requirements
+============
+
+In order to build MetalK8s we rely and third-party tools, some of them are
+mandatory, others are optional.
+
+Mandatory
+---------
+
+- `Python <https://www.python.org/>`_ 3.6 or higher: our buildchain is
+  Python-based
+- `docker <https://www.docker.com/>`_: to build some images locally
+- `skopeo <https://github.com/containers/skopeo>`_, 0.1.19 or higher: to save
+  local and remote images
+- `hardlink <https://jak-linux.org/projects/hardlink/>`_: to de-duplicate images
+  layers
+- mkisofs: to create the MetalK8s ISO
+
+Optional
+--------
+
+- `git <https://git-scm.com/>`_: to add the Git reference in the build metadata
+- `Vagrant <https://www.vagrantup.com/>`_, 1.8 or higher: to spawn a local
```

Comment:
We should convey that for now, Virtualbox is the only provider we support for the Vagrant side of things.

Reply:
Isn't that clear enough with the dependency on VirtualBox?
How can I formulate it in a clearer way?

Reply:
I was hoping you'd have an idea :)

Maybe just tack on `Virtualbox is currently the only provider supported.` to the Vagrant item?

Reply:
Done